docs/README.md

```diff
@@ -113,7 +122,7 @@ curl -sfL http://get.kubearmor.io/ | sudo sh -s -- -b /usr/local/bin
 <details><summary> Note: For distributions other than Ubuntu/Debian</summary>
 <p>
 
-i. Download release tarball from KubeArmor [releases](https://github.com/kubearmor/KubeArmor/releases)
+i. Download release tarball from KubeArmor [releases ](https://github.com/kubearmor/KubeArmor/releases){:target="_blank"}{: .externalLink}
```

Comment:
> [releases ]

[releases]

Reply:
The extra space is intentional. It will separate the linked text from the launch glyph.